install-dependencies.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Script that installs the dependencies needed to build and run Differential Dataflow
+
+echo "Installing dependencies for DDlog"
+echo "This script should be invoked with '. ./install-dependencies.sh' to set up the environment properly"
+
+case "$OSTYPE" in
+    linux*) ;;
+    *) echo "Unhandled operating system $OSTYPE"; exit 1;;
+esac
+
+
+echo "Installing Haskell"
+curl -sSL https://get.haskellstack.org/ | sh
+
+echo "Installing Rust"
+curl https://sh.rustup.rs -sSf | sh -s - -y
+export PATH=$HOME/.cargo/bin:$PATH
+
+echo "Installing Flatbuf"
```

Comment:
There is a script to install FlatBuffers in `.travis/install-flatbuf.sh`.  It works for both MacOS and Linux. Only problem is, it installs in the home directory (which makes sense on Travis, but not necessarily on the developer's machine). Your script, on the other hand, installs in the `differential-datalog` directory. I can probably change the Travis script to do the same, so we only have one script to install Travis.

Reply:
There you are worrying about travis retrying too.
It's short enough that some duplication should be OK.

Reply:
Your scripts for Travis are very different from the manual installation instructions you are giving in the README.